libs/sdk-ui-geo/src/core/geoChart/helpers/geoChart/tests/common.test.ts

```diff
@@ -178,4 +179,22 @@ describe("common", () => {
             expect(isColorAssignmentItemChanged(prevColorAssignment, colorAssignment)).toBe(true);
         });
     });
+
+    describe("getResponsiveInfo", () => {
```

Comment:
What exactly are those tests testing? What I am getting at is if they are useful and should be kept in. What do you think (we can keep them if you think we should, I was just curious about your opinion on this)?

Reply:
Removed https://github.com/gooddata/gooddata-ui-sdk/pull/1341/commits/1be0a8af4d0cfbc16189db61482d08b92f34f41f. I originally thought that it will test these differences between return string or boolean but when you wrote it I realized that typescript is sufficient in that case and I do not have to introduce unnecessary UT